stable/airflow/README.md

```diff
@@ -98,6 +98,20 @@ airflow:
     pullSecret: my-docker-repo-secret
 ```
 
+### Airflow connections
+
+Connections define how your Airflow instance connects to environment and 3rd party service providers.
+This helm chart allows you to define your own connections at the time of Airflow initialization.
+For each connection the id and the type has to be defined. All other properties are optional.
+Example:
+```yaml
+airflow:
+  connections:
+  - id: my_aws
+    type: aws
+    extra: '{"aws_access_key_id": "**********", "aws_secret_access_key": "***", "region_name":"eu-central-1"}'
```

Comment:
wonder if helm core team wont ask us to store these creds in a secret somehow.

Reply:
I agree. Trying to wrap my head around this as we speak. I would probably mean to put the whole of add-connections.sh in a secret and then mount and execute it. Instead of putting it in this configmap.